velox/exec/fuzzer/WindowFuzzer.cpp

```diff
@@ -251,7 +250,7 @@ void WindowFuzzer::go() {
     // rows by row_number additionally.
     if (requireSortedInput || isRowsFrame) {
       sortingKeysAndOrders.push_back(
-          SortingKeyAndOrder("row_number", "asc", "nulls last"));
+          SortingKeyAndOrder("row_number", core::SortOrder(true, false)));
```

Comment:
nit: SortingKeysAndOrders("row_number", kAscNullsLast);

Reply:
Updated, thanks. Could you please help take another look?